Swiftfin/Views/ConnectToServerView.swift

```diff
@@ -78,11 +78,14 @@ struct ConnectToServerView: View {
                 }
             } else {
                 Button {
+                    if !(url.contains("://")) {
+                        url = "http://" + url
+                    }
                     connectToServer()
                 } label: {
                     L10n.connect.text
                 }
-                .disabled(URL(string: url) == nil || isConnecting)
+                .disabled((URL(string: url) == nil && URL(string: "http://" + url) == nil) || isConnecting || url == "")
```

Comment:
We don't need these extra checks. `URL(string: "") == nil` and we will already be checking for a correctly formed `URL` in `ConnectToServerViewModel`.

I seem to have made this very non-functional but I can change that later. Additionally, this change should also be made on tvOS.

Reply:
Ah yes okay, I assumed `URL` would return `nil` if `http://` wasn't present